src/components/sdk/MeetingControls/docs/AudioInputVFControl.mdx

```diff
@@ -16,28 +16,55 @@ If you want to apply Amazon Voice Focus feature, try this `AudioInputVFControl`.
 
 You can use it in the [ControlBar component](/docs/ui-components-controlbar--page) to build the meeting controls bar.
 
+## Prerequisites
+
+Amazon Voice Focus and Echo Reduction features require Web Audio to be enabled in your meeting session. Configure this when joining a meeting:
+
+```javascript
+await meetingManager.join(meetingSessionConfiguration, {
+  enableWebAudio: true,
+});
+```
+
+### Voice Focus and WebAudio Best Practice
+
+When joining a meeting, only enable WebAudio when Voice Focus is both supported and desired by the user. Enabling WebAudio may cause browsers to disable their built-in noise cancellation features. If Voice Focus is not supported in this scenario, users will experience degraded audio quality due to the loss of both browser-level and SDK-level noise suppression. To prevent this, implement a conditional check: `enableWebAudio: userWantsVoiceFocus && (isVoiceFocusSupported === true)`
```

Comment:
why not use `isVoiceFocusDesired` to match the description above and the meeting demo changes.